README.md

```diff
@@ -15,3 +15,22 @@ the function and run automatically when you commit through a GitHub action.
 
 What is the worst-case big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+///
+
+When traversing the graph in union, each edge is called once. Find needs to determine whether two nodes are in the same set. The complexity depends on the number of edges E
+
+Find connects all nodes on the path to the root node each time it visits. The time complexity is O(V)
+
+Union merges two sets and calls find to determine whether the root nodes of the two nodes are the same. The time complexity of a single union is O(α(V))
+
+α is the extremely slowly growing inverse Ackermann function and the growth rate is less than log(v).
```

Comment:
Not following you here -- how did you get to the inverse Ackermann function here?